Storage/Rust/RustRemoteTabs.swift

```diff
@@ -0,0 +1,171 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+@_exported import MozillaAppServices
+
+private let log = Logger.syncLogger
+
+public extension RemoteTabRecord {
```

Comment:
nit: extensions are preferred at the end of the file so we have rust remote tabs class on top